src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,100 @@
+@use "../utils/variables" as *;
+
+.card {
+  width: 198px;
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+  box-sizing: content-box;
```

Comment:
`border-box` is more commonly used. If I were you I'd set box-sizing to this value and the `width` to 200px.